common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -161,6 +161,9 @@ div.problem {
 
     label {
       @include box-sizing(border-box);
+      @include padding($baseline/2);
```

Comment:
Great start. I think the label padding needs some adjustment so that the radio and checkbox would align up with the line height of the text. I would also reduce the spacing between the radio / checkbox and text in padding-left below.
<img width="255" alt="screen shot 2016-07-06 at 1 17 39 pm" src="https://cloud.githubusercontent.com/assets/16127998/16627632/1c6f9dc0-437c-11e6-8a68-c1a3736a3af6.png">